jenkins/test_orchestrator/JenkinsFile

```diff
@@ -0,0 +1,62 @@
+pipeline {
+    agent any
+
+    parameters {
+        string(name: 's3_bucket'),
```

Comment:
As commented below we do not need the S3 bucket parameter to be fed for the orchestrator.